rokka/client-bundle/1.0/config/packages/rokka.yaml

```diff
@@ -0,0 +1,6 @@
+rokka_client:
+    # Api-Key value given on registration of the user. Get your API key form https://rokka.io/
+    api_key:       "%env(ROKKA_API_KEY)%" # Required
+    # Organization value like "my-org" for use in calls to the API.
+    organization: "%env(ROKKA_ORGANIZATION)%" # Required
```

Comment:
@chregu is the organization not optional? i thought the bundle works without organziation and then does e.g. not register the commands that only make sense when you have an organization. that way, you can create an organization with the commands, after installing the bundle with your api key.

Reply:
Not in the bundle. It's marked as required (since ages ;)). Also for ease of use, it's good to have it there.